api/envoy/admin/v3/config_dump.proto

```diff
@@ -478,3 +478,18 @@ message EndpointsConfigDump {
   // The dynamically loaded endpoint configs.
   repeated DynamicEndpointConfig dynamic_endpoint_configs = 3;
 }
+
+// TODO: Copy this to other versions of the API
+message GenericXdsConfigDump {
```

Comment:
Please add comments for the message and every field

Reply:
Done.